### DISCUSSION TITLE
Welcome to xournalpp discussions!

### DISCUSSION THREAD

**Discussion by Technius** [xournalpp/xournalpp#2384] [General]
## GitHub Discussions (beta) now enabled for Xournal++

Hi everyone,

We're now participating in the [GitHub Discussions beta](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions). We hope you find this helpful for:
* Asking questions about features
* Sharing tips and tricks with other members of the community
* Finding out what other people are using Xournal++ for
* Getting help with installation or setting up a feature
* etc.

Of course, feel free to post anything that is not in any one of the categories we have mentioned.

As usual, if you run into a bug or would like to suggest improvements/new features, please submit a report to our [issue tracker](https://github.com/xournalpp/xournalpp/issues).

Thanks,
The Xournal++ Team

**Comment by kaiq22** [comment]
Is it possible to save a preset for a drawing? I always have to draw a trigonometric circle
